.env.example

```diff
@@ -1,2 +1,10 @@
 VITE_BACKEND_URL=
-VITE_BACKEND_WS=
\ No newline at end of file
+VITE_BACKEND_WS=
+
+VITE_FIREBASE_PROJECT_ID=
+VITE_FIREBASE_API_KEY=
+VITE_FIREBASE_AUTH_DOMAIN=
+VITE_FIREBASE_DATABASE_URL=
+VITE_FIREBASE_APP_ID=
+VITE_FIREBASE_STORAGE_BUCKET=
+VITE_FIREBASE_MESSAGING_SENDER_ID=
```

Comment:
this would work for local development, but where are we planning to store some of these secrets, for instance, API KEY?

Reply:
You mean that .env in React applications is not safe? 

Then maybe we can have a Backend endpoint to retrieve those variables. I am not sure how to protect Booster endpoints by CORS policy to allow just our frontend domain to make calls to this endpoint to get the keys

I am new to React and I am new to Booster, then I accept any proposals!

Reply:
Interesting article here about env variables and secrets
https://www.trendmicro.com/en_us/research/22/h/analyzing-hidden-danger-of-environment-variables-for-keeping-secrets.html

In any case, reading through Firebase docs
https://firebase.google.com/docs/projects/api-keys

We might be ok having the API Key exposed but we may want to apply some restrictions to Firebase

Reply:
Thank you for the resources!

I was reading the docs and I think this may help [Restrict by HTTP URL](https://cloud.google.com/docs/authentication/api-keys?_ga=2.244818179.-784472276.1673605340&hl=es-419#http)

We can restrict the use of the api key to our frontend domain

I don't know if we already have a domain for the frontend but once I know this domain I can add it to the restrictions

Reply:
nice one @melinahernandeztheam ! ❤️